docker/test/scripts/test_env

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+[ -e '/app/.env' ] && { echo 'This image does not support mounting /app/.env.' >&2; exit 1; }
```

Comment:
How about something like `Existing .env detected, please rename it temporarily before running the test container to prevent data loss.`?